DBZ-4260 Blog post for Debezium UI topic auto-creation

mdrillin opened the pull request:
blog post for topic group support in the UI

mdrillin committed:
DBZ-4260 Blog post for Debezium UI topic auto-creation

gunnarmorling commented:
@rk3rn3r, @mdrillin, it's getting late, so let's try and wrap this one up for tomorrow then. Thx!

gunnarmorling commented:
@mdrillin, hey, what's needed to undraft this one? Does it have the right video URL already? If not, @rk3rn3r, can you help with that? Let's aim for Wednesday with this, i.e. can you update the date and file name accordingly, @mdrillin?

gunnarmorling commented on _posts/2021-11-29-debezium-ui-topic-groups.adoc:
```diff
@@ -0,0 +1,47 @@
+author: mdrilling
+---
+
+The Debezium UI team is pleased to announce support for automatic creation of Topic Groups in the Debezium UI!  Our goal with the Debezium graphical user interface is to allow users to set up and operate connectors more easily.  To that end, we have added support for Topic Groups to the UI.  Read futher for more information, and for a video demo of the new feature!
```
That's rather repetitive with the previous post. Can you reword a bit, e.g. "Continuing our efforts to help users with setting up...".

gunnarmorling commented on _posts/2021-11-29-debezium-ui-topic-groups.adoc:
```diff
@@ -0,0 +1,47 @@
+
+== Automatic Topic Creation
+
+Beginning with Kafka 2.6.0, Kafka Connect comes with built-in topic creation for connector topics. If automatic topic creation is enabled, the Debezium UI now allows you to configure connector topics using the UI.
```
Here we should have a few sentences motivating topic creation settings, for those who don't know what they are good for. @rk3rn3r's [post](https://debezium.io/blog/2020/09/15/debezium-auto-create-topics/) can serve as inspiration.

gunnarmorling commented on _posts/2021-11-29-debezium-ui-topic-groups.adoc:
```diff
@@ -0,0 +1,47 @@
+We'd also be very happy to learn about your requirements and feedback on the Debezium UI.
+Please let us know in the comments below, or send a message to our https://groups.google.com/g/debezium[mailing list].
+
+_A big thank you to the team who have contributed in many ways: Ashique Ansari, Indra Shukla, René Kerner and Gunnar Morling!_
```
That's all very similar to the earlier post? Let's better avoid the duplication and either rephrase or link to the earlier post, e.g. in terms of the next features planned to be added.

gunnarmorling reviewed (commented):
LGTM overall, @mdrillin. A few comments inline.

mdrillin commented:
> @mdrillin, hey, what's needed to undraft this one? Does it have the right video URL already? If not, @rk3rn3r, can you help with that? Let's aim for Wednesday with this, i.e. can you update the date and file name accordingly, @mdrillin?

@gunnarmorling sure, I can update date and file for Wednesday and make some of the suggested text changes.  The topic group video will need to be uploaded to youtube channel and then the link updated in the article

rk3rn3r commented:
@mdrillin All videos are already on the YT channel. I shared the links on chat, back when I created the channel. You can use the link from chat. I will set the video to public when we merge the PR and publish the blog post.

mdrillin commented:
@gunnarmorling @rk3rn3r 
- renamed file and dated 2021-12-01
- Added more description content (borrowed most of the wording from documentation, etc).
- Removed some content that was repeated from other article
- updated the youtube link (sorry, I forgot the video was there but just hidden)

gunnarmorling commented:
Applied; thanks a lot, @mdrillin!
